app/javascript/App/components/ItemSelectorCategory.js

```diff
@@ -32,50 +36,51 @@ export default class ItemSelectorCategory extends Component {
       )
     } else {
       return (
-        <button onClick={() => this.add(item)}>
-          {this.itemLabel(item)}
-        </button>
+        <button onClick={() => this.add(item)}>{this.itemLabel(item)}</button>
       )
     }
   }
   render() {
     const { ItemButton } = this
 
     let colSize = 0
-    if (window.isMobile) {
+    if (this.props.isMobile) {
       colSize = 2
-    }
-    else {
+    } else {
       colSize = 3
     }
 
-    const itemGroups = this.props.items.map((x, i) => {
-      return i % colSize === 0 ? this.props.items.slice(i, i + colSize) : null;
-    }).filter(x => x != null);
-
+    const itemGroups = this.props.items
+      .map((x, i) => {
+        return i % colSize === 0 ? this.props.items.slice(i, i + colSize) : null
+      })
+      .filter((x) => x != null)
 
     return (
       <div className="item-selector-alt nh1 nt1">
         {itemGroups.map((itemGroups, index) => {
-          return (<div className="row" key={index}>
-            {itemGroups.map((item, innerIndex) => {
-              let itemLabel
-              if (item === 'Events - Conferences, Meetups, Launches, etc.') {
-                itemLabel = 'Events'
-              }
-              else if (item === 'Security (Vulnerabilities)') {
-                itemLabel = 'Security'
-              }
-              else {
-                itemLabel = item
-              }
-              return (
-                <div className="col category-btn" span={8} key={innerIndex}>
-                  <ItemButton item={itemLabel} style={{cursor:'pointer'}} />
-                </div>
-              )
-            })}
-          </div>);
+          return (
+            <div className="row" key={index}>
+              {itemGroups.map((item, innerIndex) => {
+                let itemLabel
+                if (item === 'Events - Conferences, Meetups, Launches, etc.') {
```

Comment:
I think we can safely nuke these with the release of the new NewsCategories.

Reply:
Still referenced in the `FilterPanel` by the `CalendarPage` it seems

Reply:
Ah, thanks for the catch - we can safely ignore Calendar as it basically needs to be rebuilt sans Redux anyways.